.github/workflows/main.yml

```diff
@@ -0,0 +1,181 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+
+jobs:
+  build:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ '1.16', '1.17' ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-VERSION: ${{ matrix.go }}
+      - uses: actions/cache@v2
+        id: gomod-cache
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('go.mod') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - run: go vet
+      - name: Run unit tests
+        run: go test -tags unit -race
+  integration-cassandra:
+    needs:
+      - build
+    name: Integration Tests
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        go: [ '1.16', '1.17' ]
+        cassandra_version: [ '3.0.24', '3.11.10' ]
+        auth: [ "false" ]
+        compressor: [ "snappy" ]
+        tags: [ "cassandra", "integration", "ccm" ]
+          # - cassandra_version: 3.11.10
+          #   auth: "false"
+          #   compressor: "lz4"  # panic: invalid compressor: lz4
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+      - uses: actions/cache@v2
+        id: gomod-cache
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('go.mod') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Install CCM
+        run: pip install git+https://github.com/pcmanus/ccm.git@2.8.2
```

Comment:
Should be probably github.com/riptano/ccm instead of github.com/pcmanus/ccm as the latter redirects to the former address.

Reply:
Changed. 

We should change also [install_test_deps.sh](https://github.com/gocql/gocql/blob/d83b80dfb4800e8761487e2a0c86cf6864824a8e/install_test_deps.sh)